### PR TITLE
Teletubbies/display

### PR DESCRIPTION
**Pull request by carisno**
This branch is one for displaying Teletubbies page that has ui according to the requirement using mui material and import access and map function.

### CODE REVIEW

File: src/Data/teletubbies.json
Comment (noahstoner) [inline src/Data/teletubbies.json:1]: Why did you change the path of this file?
Doesn't it work on the public folder?

File: src/pages/Teletubbies/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from "react";
+import { Container, Typography, Grid, Paper } from "@mui/material";
+import teletubbiesData from "../../Data/teletubbies.json";
```
Comment (noahstoner) [inline src/pages/Teletubbies/index.js]: I want to make the api for getting the list
So, we can use the api.js

File: src/pages/Teletubbies/index.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+const Teletubbies = () => {
+  useEffect(() => {}, []);
```
Comment (noahstoner) [inline src/pages/Teletubbies/index.js]: Why do we need this?

File: src/pages/Teletubbies/index.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+              borderColor: "primary.main",
+              color: "Blue",
+            }}
+            mt={4}
+            mb={3}
+          >
+            Teletubbies
+          </Typography>
+        </Grid>
+        {teletubbiesData.map((teletubby, index) => (
```
Comment (noahstoner) [inline src/pages/Teletubbies/index.js]: `teletubby` doesn't have an id?

File: src/pages/Teletubbies/index.js (hunk 1 above)
Comment (carisno) [reply src/pages/Teletubbies/index.js:3]: Got it. I will reset path.

File: src/pages/Teletubbies/index.js (hunk 2 above)
Comment (carisno) [reply src/pages/Teletubbies/index.js:6]: Ok. I will delete this line.

File: src/pages/Teletubbies/index.js (hunk 3 above)
Comment (carisno) [reply src/pages/Teletubbies/index.js:25]: Because there isn't in teletubbies.json.

File: src/pages/Teletubbies/index.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getTeletubbies();
+  }, []);
+
+  return (
+    <>
```
Comment (noahstoner) [inline src/pages/Teletubbies/index.js]: Why do we need empty `<>`?

**Review by shinevue — APPROVED** [review]
Approved!
